### ISSUE TITLE
ARCHIVE_DIR is statically set to a non-existant directory.

### ISSUE THREAD

**Issue by Stealthii** [pycontribs/phoebian#1]
ARCHIVE_DIR is set to `/backups/archive/`, which does not exist on most systems. This should be set to a directory by default that is available, and/or be configurable.
##

**Comment by ssbarnea** [comment]
I totally agree with you, should be easily fixable by checking if the env variable is set, if not to keep the current behaviour. Also, I do welcome pull requests! ;)
